chacractl/tests/test_main.py

```diff
@@ -25,6 +25,6 @@ def test_ssl_verify(self, tmpdir, monkeypatch, template, expected):
         config_path = tmpdir.join('.chacractl')
         config_path.write(template)
         monkeypatch.setattr(os.path, 'expanduser', lambda _: str(config_path))
-        c = ChacraCtl()
+        c = ChacraCtl('repo')
```

Comment:
the arg here is supposed to be a `list`, so I'd argue against using a string even if it's just for the test